proto/osmosis/poolmanager/v1beta1/query.proto

```diff
@@ -24,14 +24,30 @@ service Query {
   rpc EstimateSwapExactAmountIn(EstimateSwapExactAmountInRequest)
       returns (EstimateSwapExactAmountInResponse) {
     option (google.api.http).get =
-        "/osmosis/gamm/v1beta1/{pool_id}/estimate/swap_exact_amount_in";
+        "/osmosis/poolmanager/v1beta1/{pool_id}/estimate/swap_exact_amount_in";
+  }
+
+  rpc EstimateSinglePoolSwapExactAmountIn(
+      EstimateSinglePoolSwapExactAmountInRequest)
+      returns (EstimateSwapExactAmountInResponse) {
+    option (google.api.http).get =
+        "/osmosis/poolmanager/v1beta1/{pool_id}/estimate/"
+        "single_pool_swap_exact_amount_in";
```

Comment:
oof just noticed that this is automatically done by linter, should not be divided by lines, or else would not work

Reply:
I assumed it automatically concatenated adjacent strings (some other languages do that). Is that not the case? it seems like the proto generation works

Reply:
changed to one line just in case

Reply:
thought so as well, but I don't think proto compilers support this tho 😭